devito/interfaces.py

```diff
@@ -4,11 +4,12 @@
 from tools import aligned
 import os
 from signal import signal, SIGABRT, SIGINT, SIGSEGV, SIGTERM
+from tempfile import gettempdir
 import sys
 import atexit
 
 
-__all__ = ['DenseData', 'TimeData', 'PointData']
+__all__ = ['DenseData', 'TimeData', 'PointData', 'MemmapManager', 'SymbolicData']
```

Comment:
Hi, sure, we will change this back once we are happy with the documentation. The reason why the are here is because Sphinx does not generate doc for these classes if they are not in __all__, may be there are ways to tell it to ignore __all__, but this seems like a quick fix just to generate doc. This change will not be merged.